specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -318,14 +318,14 @@ union TestKind {
   string,
 
   @doc("URL Test")
-  "URL",
+  Url: "URL",
```

Comment:
URL? Would these come up as Url or URL in clients?

Reply:
Discussed offline